frontend/src/assets/Auth modal graphic.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="461" height="229" viewBox="0 0 461 229" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Unstage this from this PR. We need either a graphic the same size as the modal, or a better quality graphic which we custom position in the modal